src/aat/java/uk/gov/hmcts/cmc/claimstore/tests/functional/RespondToClaimTest.java

```diff
@@ -39,17 +40,20 @@ public void shouldBeAbleToSuccessfullySubmitAlreadyPaidDefence() {
     }
 
     private void shouldBeAbleToSuccessfullySubmit(Response response) {
+        String claimantId = functionalTestsUsers.getClaimant().getUserDetails().getId();
         Claim createdCase = commonOperations.submitClaim(
             functionalTestsUsers.getClaimant().getAuthorisation(),
-            functionalTestsUsers.getClaimant().getUserDetails().getId()
+            claimantId
         );
 
-        User defendant = idamTestService.createCitizen();
+
+        String letterHolderId = createdCase.getLetterHolderId();
+        Integer letterHolderUserId = StringUtils.isNotBlank(letterHolderId) ? Integer.valueOf(letterHolderId)
```

Comment:
@dharmendrak, I had a chat with @timja-kainos about this. We can accept this tech debt short term when we proved it works for us locally (not only you) and it is not failing on CNP.

Long term we will spark changes in CCD to lets us:
- say who we grant access to e.g. claimant, letter holder, defendant
- query who has access to the particular case